lib/widgets/pylons_button.dart

```diff
@@ -1,19 +1,29 @@
 import 'package:easel_flutter/utils/easel_app_theme.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
 
 class PylonsButton extends StatelessWidget {
   final VoidCallback onPressed;
-  const PylonsButton({Key? key, required this.onPressed}) : super(key: key);
+  final String btnText;
+  final bool showArrow;
+
+  const PylonsButton(
+      {Key? key,
+      required this.btnText,
+      required this.showArrow,
+      required this.onPressed})
+      : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return TextButton.icon(
+    return /*TextButton.icon(
```

Comment:
What's the purpose of leaving all of this commented? Is this a to-do later kind of item?

Reply:
Forgot to remove them. thanks for pointing this out.